jenkinsapi/jenkins.py

```diff
@@ -39,7 +39,7 @@ def __init__(
             self, baseurl,
             username=None, password=None,
             requester=None, lazy=False,
-            ssl_verify=True, cert=None, timeout=10):
+            ssl_verify=True, cert=None, timeout=10, use_baseurl=False):
         """
         :param baseurl: baseurl for jenkins instance including port, str
```

Comment:
Should there be docstrings for the `use_baseurl`? And perhaps we can add them for the others too?

Reply:
Done